contrib/freshmaker-cli

```diff
@@ -225,6 +250,53 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url, days=7):
+    """
+    Sends the GET request to the preset Errata filter. Filter shows RHSA in SHIPPED_LIVE state
+    ordered by date of shipping (from the latest to the oldest). Returns the list of advisories
+    that are IMPORTANT or CRITICAL and the date of shipping is older given number of days before the
+    current date.
+    """
+    if days > 7:
+        print("The interval between dates should not exceed 7 days")
+        sys.exit(1)
+    page = 1
+    while True:
+        print("Getting advisories from  Errata page", page)
+        krb_auth = HTTPKerberosAuth()
+        r = requests.get(errata_url, auth=krb_auth, timeout=30, params={page: page})
+        r.raise_for_status()
+        data = r.json()
+        advisories = []
+        date_from = date.today() - timedelta(days)
+        for advisory in data:
```

Comment:
We should guard against the Errata Tool returning no advisories on the page we are querying.
```suggestion
        if not data:
            return advisories
        for advisory in data:
```